stm32-modules/heater-shaker/firmware/system/system_policy.hpp

```diff
@@ -1,6 +1,29 @@
 #pragma once
 
+#include <array>
+
+#include "heater-shaker/errors.hpp"
+#include "systemwide.hpp"
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wvolatile"
+#include "system_hardware.h"
+#include "system_serial_number.h"
+#pragma GCC diagnostic pop
+
 class SystemPolicy {
+  private:
+    static constexpr std::size_t system_serial_number_length =
```

Comment:
Can we make these constants `NAMED_LIKE_THIS` so it's a bit more apparent in the moment that they're constants